### DISCUSSION TITLE
Unwanted "booked" tag after nordigen import

### DISCUSSION THREAD

**Discussion by jlauwers** [firefly-iii/firefly-iii#8276] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Title :-)
It look's like a tag used for debug?

### Debug information

Using fireflyiii/data-importer:latest

### Expected behaviour

No tag?

**Comment by JC5** [comment]
Part of the new importer, so it's here to stay. Nordigen can create both "pending" and "booked" transactions and this tag allows you to see the difference.

You can create a rule to remove it.

**Reply by jlauwers** [reply]
So, I guess the importer will change the tag when the transaction is changing from pending to booked?

**Reply by JC5** [reply]
No. The tag stays the same. Usually pending > booked transactions are detected as duplicates so not re-imported, OR pending transactions have a lot of blanks so users can use the tag to delete them

**Reply by alexchatwin** [reply]
Hi JC5 - I think this behaviour is leaving me with pending transactions imported to my account which never get updated with the full info once the transaction clears - this feels suboptimal. Is the work-around to never import 'pending'?

**Reply by alexchatwin** [reply]
I could make a rule to delete them, then trigger a purge from the api before my scheduled import task - I'd be grateful if you had a less hacky-approach!

**Reply by JC5** [reply]
Yes! Create a rule that triggers when the tag is "pending", and then delete the transaction.

**Comment by jaska087** [comment]
I have not yet seen pending transactions, only booked. I do keep getting duplicate transactions more often now and I need to fix everything manually. Should I just auto delete every transactions tagged as booked?

**Reply by JC5** [reply]
That depends on your duplicate detection settings. If you create such a rule, I expect no transactions will ever be imported.

**Reply by jaska087** [reply]
I think it was the default (classic) method. I need to re-run the importer again and see if something has changed, it should still be valid for 1,5 month or so.

E: Ooh.. so importer now shows available, cleared, booked balance, does firefly iii use any of this info? Should I just untick import tags? my bank does not provide tags, so I just use description/amount based rules.

EE:  Has this change happened recently? should I expect it to work again once the threshold (for me, import last 7 days) is over?

**Reply by JC5** [reply]
> Ooh.. so importer now shows available, cleared, booked balance, does firefly iii use any of this info?

No, Firefly III doesn't use the bank's balance, it calculates the balance itself using the provided transactions.

> Should I just untick import tags? 

The "booked" tag will always be imported from GoCardless, regardless of this setting.

>  Has this change happened recently? should I expect it to work again once the threshold (for me, import last 7 days) is over?

Yes, in 1.4.0. It should clear itself after a few days, as you say.
